pkg/testers/ginkgo/ginkgo.go

```diff
@@ -155,6 +170,29 @@ func (t *Tester) validateLocalBinaries() error {
 	return nil
 }
 
+// ginkgoMajorVersion returns the ginkgo major version
+// empty if not found
+func (t *Tester) ginkgoMajorVersion() string {
+	klog.V(2).Infof("checking ginkgo version ...")
+	cmd := exec.Command(t.ginkgoPath, "version")
+	lines, err := exec.OutputLines(cmd)
+	if err != nil || len(lines) != 1 {
+		return ""
+	}
+	// the output is in the format
+	// Ginkgo Version 1.14.0
+	// Ginkgo Version 2.1.4
+	parts := strings.Split(lines[0], " ")
+	if len(parts) != 3 {
+		return ""
+	}
+	vers := strings.Split(parts[2], ".")
+	if len(vers) != 3 {
```

Comment:
not sure if ginkgo will has a version like "2.x.x.x" or "2.x" in the future, is the major parsed from the string is enough?